# Урок 4. Клинические исследования

## Домашнее задание.

Найти на сайте clinicaltrials.gov с помощью расширенного поиска interventional исследование, лучше 3 фазы, у которого будет доступен план статистического анализа (SAP) или протокол и результаты.
1. Посмотреть как устроен документ
2. Найти описание дизайна исследования - рандомизированное или нет, какое заслепление, какое показание, какие группы сравнения, сколько человек в каждой группе сравнения, есть ли стратификация и по каким признакам
3. Найти раздел с конечными точками, разобрать каждую точку и определить ее тип - первичная/вторичная, клиническая/суррогатная
4. Посмотреть результаты на сайте на странице исследования, определить было исследование успешным или нет.


In [1]:
import pandas as pd
import numpy as np
import os
# import tarfile
import pprint
import seaborn as sns 
import matplotlib.pyplot as plt
%matplotlib inline

In [2]:
!chcp 65001

Active code page: 65001


In [9]:
pd.set_option('display.max_rows', None)

### Описание выбранного исследования.
Рассмотрим исследование **`NCT04676412`** *Efficacy and Safety Study of Pembrolizumab (MK-3475) With or Without Lenvatinib (MK-7902/E7080) in Adults With Programmed Cell Death-Ligand 1 (PD-L1)-Positive Treatment-naïve Nonsmall Cell Lung Cancer (NSCLC) [MK-7902-007/E7080-G000-314/LEAP-007] - China Extension Study (LEAP-007)*  
*__[Исследование эффективности и безопасности терапии Пембролизумабом (MK-3475) с или без комбинации с Ленватинибом (MK-7902/E7080) у взрослых PD-L1-позитивных пациентов с немелкоклеточным раком лёгкого (НМРЛ) ранее не получавших лечение (Расширенное исследование для территории Китая)]__*  
Спонсором данного исследования является компания *Merck Sharp & Dohme (MSD)*  
Ссылка на страницу исследования: [NCT04676412](https://clinicaltrials.gov/study/NCT04676412?aggFilters=ages:adult,docs:sap,phase:3,results:with&distance=50&viewType=Table&limit=100&page=1&rank=92&tab=results#outcome-measures "NCT04676412")

In [4]:
os.listdir('20_4_files')

['NCT04676412.csv']

In [10]:
trial = pd.read_csv('20_4_files/NCT04676412.csv', sep=',')
trial.T

0
NCT Number                                                        NCT04676412
Study Title                 Efficacy and Safety Study of Pembrolizumab (MK...
Study URL                        https://clinicaltrials.gov/study/NCT04676412
Acronym                                                              LEAP-007
Study Status                                            ACTIVE_NOT_RECRUITING
Brief Summary               The purpose of this study is to assess the saf...
Study Results                                                             YES
Conditions                                         Non-small Cell Lung Cancer
Interventions               BIOLOGICAL: Pembrolizumab|DRUG: Lenvatinib|DRU...
Primary Outcome Measures    Progression-free Survival (PFS) Per Response E...
Secondary Outcome Measures  Objective Response Rate (ORR) Per RECIST 1.1 a...
Other Outcome Measures                                                    NaN
Sponsor                                               Merck Sharp & Dohme LLC
Collaborators                                                      Eisai Inc.
Sex                                                                       ALL
Age                                                        ADULT, OLDER_ADULT
Phases                                                                 PHASE3
Enrollment                                                                 27
Funder Type                                                          INDUSTRY
Study Type                                                     INTERVENTIONAL
Study Design                Allocation: RANDOMIZED|Intervention Model: PAR...
Other IDs                   7902-007 China Extension|MK-7902-007|E7080-G00...
Start Date                                                         2019-10-23
Primary Completion Date                                            2021-05-19
Completion Date                                                    2023-11-30
First Posted                                                       2020-12-21
Results First Posted                                               2022-10-14
Last Update Posted                                                 2022-11-09
Locations                   Anhui Provincial Hospital ( Site 0108), Hefei,...
Study Documents                      Study Protocol|Statistical Analysis Plan

### 1. Описание документа исследования  
Документ выбранного исследования состоит из 4 вкладок:
 1. Детали исследования (Study Details)
 2. Основные сведения в виде таблицы (Table View)
 3. Опубликованные результаты (Results Posted)
 4. История изменений (Record History)   
  
Вкладка Детали исследования (Study Details) содержит основные данные об исследовании:
 1. Обзор исследования (Study Overview)
     - краткое описание исследования
     - детальное описания исследования
     - официальное название
     - состояние (болезнь) при которой проводится это исследование
     - исследуемое вмешательство/лечение
     - ID данного исследования в других базах 
 2. Центры проведения исследования и их контакты (Contacts and Locations)
 3. Критерии участия (Participation Criteria)
     - критерии включения пациентов в исследование
     - критерии ислючения пациентов из исследования
 4. План ислледования (Study Plan) -  в этом разделе кратко описан дизайн исследования и конечные точки
 5. Коллаборации и исследователи (Collaborators and Investigators) - раздел в котором перечислены вовлеченные в исследование организации и исследователи
 6. Публикации (Publications)
 7. Временные рамки исслелования (Study Record Dates) - даты начала/окончания набора пациентов/наблюдений и другие важные для исследования даты
 8. Дополнительная информация (More Information)
 
Вкладка Опубликованные результаты (Results Posted) частично повторяет сведения из вкладки Детали исследования (Study Details), в частности содержит разделы *Study Record Dates, Collaborators and Investigators, Study Record Dates* и *More Information*.  
Отличающиеся разделы:
 - Поток участников (Participant Flow) - в этом разделе описаны измениния в когортах пациентов (сколько пациентов стартовало/закончило исследование, сколько выбытло по различным причинам)
 - Стартовые характеристики (Baseline Characteristics) - в этом разделе описаны исследуемые характеристики (размер опухоли/показатели крови/давления и т.п.) на момент начала исследования
 - Критерии оценки (Outcome Measures) - раздел в котором описаны конечные точки и методы их оценки
 - Неблагоприятные события (Adverse Events) - подробно описываются нежелательные явления, возникшие вследствие проводимого лечения
 - Ограничения и предостережения (Limitations and Caveats)  
  
Вкладка История изменений (Record History) содержит все изменения, проходившие с записью (старт/стоп исследования, внесение результатов, начало/конец набора пациентов и т.п.)

### 2. Дизайн исследования  
  
**`NCT04676412`** - многоцентровое, рандомизированное, тройное слепое (Participant, Investigator, Outcomes Assessor) исследование.  
2 параллельных группы в разных лечебных учреждениях на территории Китая. Все пациенты взрослые с немелкоклеточным раком лёгкого 4 стадии, позитивные по PD-L1, без метастазов в ЦНС:  
 - группа A: 49 пациентов получает Пембролизумаб внутривенно 200 мг каждый 21-й день начиная с 1-го применения + Ленванитиниб 20 мг перорально 1 раз в день каждый день. Схема терапии действует до прогрессирования заболевания, либо непереносимой токсичности.  
 - группа B: 58 пациентов получает Пембролизумаб внутривенно 200 мг каждый 21-й день начиная с 1-го применения + плацебо перорально 1 раз в день каждый день. Схема терапии действует до прогрессирования заболевания, либо непереносимой токсичности.   
  
Длительность исследования рассчитана на 35 инъекций Пембролизумаба (приблизительно 2 года).
Средний возраст пациентов в обоих группах 64,2 года. 
  
Стратификация:
 - по признаку пола
 - по статусу ECOG
 - по уровню экспрессии PD-L1

### 3. Конечные точки
Превичная клиническая конечная точка исследования - Выживаемость без прогрессирования, оценка по критериям RECIST 1.1.  
Вторичные точки:
 1. клинические: 
     - Общая выживаемость
     - Выживаемость без прогрессирования
     - Частота объективного ответа
     - Количество пациентов с нежелательными явениями
     - Количество пациентов досрочно прекративших терапию вследствие нежелательных явлений
 2. суррогатные:
     - изменение от исходного в общей оценке здоровья пациентом (баллы по опроснику EORTC QLQ-C30 Item 29)
     - изменение от исходного в оценке качества жизни пациентом (баллы по опроснику EORTC QLQ-C30 Item 30)
     - изменение от исходного в оценке кашля пациентом (баллы по опроснику EORTC QLQ-LC13 Item 31)
     - изменение от исходного в оценке боли в груди пациентом (баллы по опроснику EORTC QLQ-LC13 Item 40)
     - изменение от исходного в оценке одышки пациентом (баллы по опроснику EORTC QLQ-C30 Item 8)
     - изменение от исходного в оценке физической функциональности пациентом (баллы по опроснику EORTC QLQ-C30 Items 1-5)
     - время до истинного ухудшения общей оценки здоровья пациентом (баллы по опроснику EORTC QLQ-C30 Item 29)
     - время до истинного ухудшения оценки качества жизни пациентом (баллы по опроснику EORTC QLQ-C30 Item 30)
     - время до истинного ухудшения комбинированной конечной точки по показателям оценки кашля и боли в груди или одышки пациентом (баллы по опросникам EORTC QLQ-LC13 Items 31 & 40 или EORTC QLQ-C30 Item 8)
     - время до истинного ухудшения оценки физической функциональности пациентом (баллы по опроснику EORTC QLQ-C30 Items 1-5)

### 4. Результаты исследования  

На текущий момент опубликованы только промежуточные результаты исследования и обработка данных еще продолжается. За время ислледования в группе A умерло 17 пациентов, в группе B 23 пациента. Ещё 2 пациента из группы B досрочно прекратили терапию самостоятельно. Полностью завершили исследование 32 пациента в группе A и 33 пациента в группе B. Анализу подверглись данные 42 и 53 пациентов в группах A и B соответственно. Медиана выживаемости без прогрессирования в группе Пембролизумаб+Ленванитиниб составила 6,1 мес против 10,3 мес в группе Пембролизумаб+плацебо. Частота объективного ответа в группе Пембролизумаб+плацебо также была выше: 39,6% против 33,3% в группе Пембролизумаб+Ленванитиниб. На ооснове этих данных исследование признано провалившимся.

__*Комментарии преподавателя:*__<br>
*Дарья Бородко・Преподаватель<br>*

>*Михаил, добрый вечер!  
Мне от вас нужно нечто большее, чем табличка из NCBI :) Да, она содержит практически все, что нужно, но не понятна, например, стратификация, если она вообще есть. Как классифицируются точки клинические/суррогатные? По какому принципу набирали участников? Каковы результаты исследования и есть ли они вообще?
Пока это только хор*
  
**Обновлено**  
*Дарья Бородко・Преподаватель<br>*

>*Михаил, добрый вечер!  
Стало совсем хорошо :)*